templates/etc/systemd/system/notification.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Notification Service
```

I would like for us to include the application name in the description, so that it's easier to find Permanent-specific services via `systemctl list-units | grep -i permanent` or so. I hadn't noticed that we did not do this with the upload service, but [we are with our other services](https://bitbucket.org/permanent-org/back-end/commits/2c64069378dfb0a6a1eafc9cdac38e5705ee6f7b).

```suggestion
Description=Permanent.org Notification Service
```